src/chatterbox_ranch_protocol.erl

```diff
@@ -0,0 +1,49 @@
+-module(chatterbox_ranch_protocol).
+
+-include("http2_socket.hrl").
+%%-behaviour(ranch_protocol).
```

Comment:
Yes! I did not want to include `ranch` as a dependency. The behaviour still works as designed, but you can't actually declare the directive.